scripts/build/build.sh

```diff
@@ -24,10 +38,26 @@ source /etc/profile.d/rvm.sh
 
 echo "current dir: $(pwd)"
 
+if [ -d "dist" ]; then
+  rm -rf dist
+fi
+
 if [ "$CIRCLE_TAG" != "" ]; then
+  echo "Building frontend from tag $CIRCLE_TAG"
+  go run build.go -buildNumber=${CIRCLE_BUILD_NUM} -includeBuildNumber=false build-frontend
   echo "Packaging a release from tag $CIRCLE_TAG"
-  go run build.go -buildNumber=${CIRCLE_BUILD_NUM} -includeBuildNumber=false package latest
+  go run build.go -goos linux -pkg-arch amd64 -buildNumber=${CIRCLE_BUILD_NUM} -includeBuildNumber=false package-only latest
+  go run build.go -goos linux -pkg-arch armv7 -buildNumber=${CIRCLE_BUILD_NUM} -includeBuildNumber=false package-only
```

Comment:
this will create deb package with armv7 as architecture (build log on circle ci show `fpm ... -a armv7 ...`. But on debian, there is no armv7 arch, only armhf. This will produce error with standard install and due to dependencies on adduser and libfontconfig, a simple `dpkg --add-architecture` won't be enough.

Reply:
Thanks for letting us know. I'll remove that from the build.